docker/development/deploy-unlock.js

```diff
@@ -14,12 +14,8 @@ const TokenDeployer = require('./deploy-locks')
  */
 const host = process.env.HTTP_PROVIDER_HOST
 const port = process.env.HTTP_PROVIDER_PORT
-let deployedLockAddress
-let purchaser = process.env.PURCHASER_ADDRESS
-let recipientAddress = process.env.ERC20_TOKEN_RECIPIENT
-let bootstrapTransferAmount = process.env.BOOTSTRAP_AMOUNT
-let bootstrapTranferRecipient = process.env.ETHEREUM_ADDRESS
-let contractOwnerAddress = process.env.CONTRACT_OWNER_ADDRESS
+let purchaser = process.env.LOCKSMITH_PURCHASER_ADDRESS // This is the locksmith user account
+let userAdress = process.env.ETHEREUM_ADDRESS // This is a user account
```

Comment:
Would you mind if we adjusted the spelling here and where this is referenced, also what is meant by user account here?

Reply:
What do you call adjust the spelling?